pkg/kudoctl/util/kudo/install.go

```diff
@@ -63,6 +64,26 @@ func InstallPackage(kc *Client, resources *packages.Resources, skipInstance bool
 		if _, err := kc.InstallInstanceObjToCluster(resources.Instance, namespace); err != nil {
```

Comment:
I think I would rather the logic for this "wait" to be included in the client so that people using the client to install can leverage this.   Eventually we would have the update call also have a "wait" option

Reply:
I had to add my own "wait" function in the TF provider since there wasn't one in the client. Adding this wait like you've done won't let me eliminate the wait I have:

https://github.com/runyontr/terraform-provider-kudo/blob/master/kudo/resource_kudo_instance.go#L547-L576